### DISCUSSION TITLE
Possible to assign "key release" to a keycode under the condition no other keys have been pressed?

### DISCUSSION THREAD

**Discussion by mitchmindtree** [vial-kb/vial-gui#95] [Q&A]
Hi folks, thanks for this project!

I'm switching from my old Anne Pro 2 to a custom Freebird build a friend made for me. I've manage to map everything how I used to have it (with some improvements), however the last mappings I'm missing are some key release mappings

On the Anne Pro 2, the release of the R shift, R ctrl, R menu and R gui buttons map to Up, Right, Down and Left respectively, but only under the condition that no other key was pressed (i.e. it hasn't modified anything) since the key was pressed. As these are 60%ers, I used to find this behaviour super handy for scrolling up commands in the terminal while sipping a coffee :)

Any ideas on how I might go about achieving this behaviour with Vial?

**Comment by lesshonor** [comment]
Sounds like you're looking for [Mod-Tap keycodes](https://docs.qmk.fm/#/mod_tap).
